CRM/Core/Error.php

```diff
@@ -334,7 +334,8 @@ public static function fatal($message = NULL, $code = NULL, $email = NULL) {
 
     if (php_sapi_name() == "cli") {
       print ("Sorry. A non-recoverable error has occurred.\n$message \n$code\n$email\n\n");
-      debug_print_backtrace();
+      // Fix for CRM-16899
+      //debug_print_backtrace();
```

Comment:
IMHO, it's a good to display a backtrace in this context (ie when encountering an error in command-line scripts). Simplest work-around:

```
echo static::formatBacktrace(debug_backtrace());
```

Alternatively, there may be a way to take advantage of PHP's `display_errors` option, but that's more fiddly.